### ISSUE TITLE
Garden cloche block duplication

### ISSUE THREAD

**Issue by bluelightning32** [SpongePowered/SpongeForge#2700]
**I am currently running**
- SpongeForge version: spongeforge-1.12.2-2768-7.1.6-RC3665.jar
- Forge version: forge-1.12.2-14.23.5.2822-universal.jar
- Java version: 1.8.0_201
- Operating System: Fedora 29 64bit

- Plugins/Mods:
```
[20:41:45] [Server thread/INFO] [minecraft/DedicatedServer]: Plugins (9): Minecraft, Minecraft Coder Pack, SpongeAPI, Sponge, Inventory Sync, JdaLibraryLoader,
LuckPerms, UltimateChat, spark
```

- Modpack: DD&SS 4.9, which includes ImmersiveEngineering-0.12-89.jar

I tried removing the Auto Pickup mod (which comes with DD&SS). It doesn't make a difference for the repro.

**Issue Description**
Repro steps:
1. Place down a garden cloche.
2. The garden cloche is 3 blocks high. Break the top block of the garden cloche. It drops 1 garden cloche.
3. Break the remaining 2 blocks of the garden cloche (they should have disappeared when the first block broke). It drops 2 more garden cloches.

This is a screenshot of after the top 2 blocks have been broken. Notice how the block outline is around the bottom block, even though it should be around the middle block (if it still existed):
![image](https://user-images.githubusercontent.com/21202621/56452171-6babd680-62e4-11e9-82fe-9fa13c7a5033.png)

I also tried this with the IE coke oven. The coke oven disappears after the first block is broken. It leaves behind invisible blocks for all the other positions. None of the coke oven blocks drop anything when broken.

**Comment by bluelightning32** [comment]
Does not reproduce in spongeforge-1.12.2-2768-7.1.6-RC3629.jar.

**Comment by Shybella** [comment]
Also encountering this on RC3665 from 3616.

**Comment by phit** [comment]
please retest with 7.1.6-RC3683

**Comment by bluelightning32** [comment]
I tested spongeforge-1.12.2-2768-7.1.6-RC3684.jar. It's much better, but not completely fixed.

1. Place down garden cloche.
2. Break the top block of the garden cloche. The garden cloche is dropped.
- Visually the top and middle blocks are gone, but one can pass through those blocks now.
- The bottom block remains and one can stand on it.
- Breaking the bottom block does not drop a second garden cloche.

My guess is that the bottom block is still there because the client isn't synchronized properly with the server.
- Right clicking on the phantom bottom block causes it to disappear.
- Leaving the area and coming back causes all the phantom blocks to disappear.

**Comment by bluelightning32** [comment]
Confirmed fixed in spongeforge-1.12.2-2825-7.1.6-RC3692.jar.
